src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
```

Comment:
You Use for loop for creating several objects of the same class and to avoid code duplication for the printing method. Steps to refactor the code:

1. Create an array of the Lottery objects with specified size - 3 in our case
2. Create for loop which 
      a.  Creates Lottery object
      b.  Prints ball information